catalog.html

```diff
@@ -1,271 +1,284 @@
+          </div>
+        </li>
+        <li class="product-card">
+          <a class="product-card-link" href="#">
+            <img class="product-card-image" src="" alt="">
+            <h3 class="product-card-title">Desert Quail Inn</h3>
+          </a>
+          <p class="miniblock-left">Мотель</p>
+          <p class="miniblock-right">От 3000 ₽</p>
+          <a href="#" class="product-card-button button">Подробнее</a>
+          <a href="#" class="product-card-button button">В избранное</a>
+          <div class="rating">
+            <p class="rating-stars">
+              <span class="visually-hidden">Мотель 3 звезды</span>
+            </p>
+            <p class="rating-text">Рейтинг: 8,9</p>
+          </div>
+        </li>
+        <li class="product-card">
+          <a class="product-card-link" href="#">
+            <img class="product-card-image" src="" alt="">
+            <h3 class="product-card-title">Villas at Poco Diablo</h3>
+          </a>
+          <p class="miniblock-left">Апартаменты</p>
+          <p class="miniblock-right">От 2000 ₽</p>
+          <a href="#" class="product-card-button button">Подробнее</a>
+          <a href="#" class="product-card-button button">В избранное</a>
+          <div class="rating">
+            <p class="rating-stars">
+              <span class="visually-hidden">Апартаменты 2 звезды</span>
+            </p>
+            <p class="rating-text">Рейтинг: 9,2</p>
+          </div>
+        </li>
+        <li class="product-card">
+          <a class="product-card-link" href="#">
+            <img class="product-card-image" src="" alt="">
+            <h3 class="product-card-title">GreenTree Inn</h3>
+          </a>
+          <p class="miniblock-left">Гостиница</p>
+          <p class="miniblock-right">От 3000 ₽</p>
+          <a href="#" class="product-card-button button">Подробнее</a>
+          <a href="#" class="product-card-button button">В избранное</a>
+          <div class="rating">
+            <p class="rating-stars">
+              <span class="visually-hidden">Гостиница 2 звезды</span>
+            </p>
+            <p class="rating-text">Рейтинг: 9,2</p>
+          </div>
+        </li>
+      </ul>
+      <button class="button pagination-show-more" type="button">Загрузить еще</button>
+    </section>
+    <div class="pagination">
+      <ul class="pagination-list">
+        <li class="pagination-item">
+          <a class="pagination-link pagination-current">1</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">2</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">3</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">4</a>
+        </li>
+        <li class="pagination-item">
+          <a class="pagination-link" href="#">...
+            <span class="visually-hidden">Переход на 5 страницу</span>
+          </a>
```

Comment:
Лучше не комбинировать содержание то на той же строке, то на отдельных.
Я на отдельных предпочитаю:

```suggestion
          <a class="pagination-link" href="#">
            ...
            <span class="visually-hidden">Переход на 5 страницу</span>
          </a>
```

Reply:
исправила